generators/client/files-angular.js

```diff
@@ -101,7 +101,8 @@ const files = {
                 { file: '_favicon.ico', method: 'copy' },
                 '_robots.txt',
                 '_404.html',
-                '_index.html'
+                '_index.html',
+                '_manifest.webapp'
```

Comment:
I would recommend renaming this file to `manifest.json` since it is a JSON file. Also, Ionic uses convention.